src/caps/parse/parse.cpp

```diff
@@ -139,175 +139,6 @@ bool my_plugin::parse_container_json_2_event(
     return true;
 }
 
-std::string my_plugin::compute_container_id_for_thread(const falcosecurity::table_entry& thread_entry,
```

Comment:
Moved to plugin.cpp since they must be available even if PARSE cap is not built in.